change/@fluentui-react-teaching-popover-b32cb5ff-1e29-4c94-bb2d-a5ad448d78f9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "visual bug fix",
```

Comment:
We're usually a little more verbose in our changelog comments.

There's a little more info here, 
https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/dev-workflow.md#making-a-pull-request

But maybe draft off the changelog in react-components for more.

Reply:
Updated it through rebase. I'm not sure how to re-run `yarn change` such that I can edit this. Can I manually edit it?

Reply:
You can modify the change file it made already directly, something like 'Fix: Arrow was 1px offset and incorrect box sizing'